vscode-tas-client/src/vscode-tas-client/VSCodeTasClient.ts

```diff
@@ -64,7 +63,7 @@ export function getExperimentationService(
         telemetry: telemetry,
         storageKey: storageKey,
         keyValueStorage: keyValueStorage,
-        featuresTelemetryPropertyName: featuresTelemetryPropertyName,
+        featuresTelemetryPropertyName: '',
```

Comment:
Will drop this once we update the `tas-client` dependency to a version where this property is optional.